src/site/filters/liquid.js

```diff
@@ -61,6 +73,41 @@ module.exports = function registerFilters() {
     return `https://img.youtube.com/vi/${string}/sddefault.jpg`;
   };
 
+  liquid.filters.outputLinks = data => {
```

Comment:
Do you have an example of what the data looks like for this?

Reply:
You bet, @jbalboni - it's just plain text: ```Email us: metrowebz@gmail.com
www.google.com
http://google.com
941-915-1811```
then after filter treatment: ```Email us: <a target="_blank" href="mailto:metrowebz@gmail.com">metrowebz@gmail.com</a>
<a href="http://www.google.com" target="_blank" rel="noopener ">www.google.com</a>
<a href="http://google.com" target="_blank" rel="noopener ">http://google.com</a>
<a href="tel:941-915-1811">Phone: 941-915-1811</a>```

Reply:
Updated per convo in slack to just use phone search. Field is now rich text, @jbalboni